.gitignore

```diff
@@ -1,3 +1,4 @@
 composer.lock
 vendor
-bin
\ No newline at end of file
+bin
+.idea
```

Comment:
@mplachta Could you remove `.idea` from ignore list? It's not related to bundle, I think it is better to add to the local gitignore.